assets/company-admin/components/CompanyAdminApp.jsx

```diff
@@ -117,7 +117,7 @@ class CompanyAdminAppComponent extends React.Component {
                         >
                             <button
```

Comment:
@petrjasek Should we comment out this button? At the moment we have the CompanyAdmin sidebar with 1 entry. Removing the button now until we add more pages to CompanyAdmin?

Reply:
yep that would make sense

Reply:
done